akka-actor/src/main/scala/akka/io/Tcp.scala

```diff
@@ -431,7 +431,23 @@ object Tcp extends ExtensionId[TcpExt] with ExtensionIdProvider {
    * Whenever a command cannot be completed, the queried actor will reply with
    * this message, wrapping the original command which failed.
    */
-  final case class CommandFailed(cmd: Command) extends Event
+  final case class CommandFailed(cmd: Command) extends Event {
+    private var _cause: Option[Throwable] = None
```

Comment:
we don't have any proper serializer for them (they are intended to be used locally), but adding `@transient` seems like a good precaution